### ISSUE TITLE
Panic when Prometheus unexpectedly unavailable 

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-analyzer-helm-chart#400]
```
I0421 05:09:08.661900       1 profiling.go:14] [Profiler] 31.834766138s: ComputeAggregateCostModel(duration=1d, offet=1m, field=namespace)
panic: interface conversion: interface {} is nil, not map[string]interface {}
goroutine 318 [running]:
github.com/kubecost/cost-model/pkg/costmodel.NewQueryResults(0x0, 0x0, 0xc0007bdc18, 0x75935b, 0x34d8e80, 0xc000000000, 0xc00051eee0)
	/app/cost-model/pkg/costmodel/promparsers.go:61 +0xb26
github.com/kubecost/cost-model/pkg/costmodel.getNormalizations(0x0, 0x0, 0x34d88a0, 0x19, 0xc0007be470, 0x1, 0x1)
	/app/cost-model/pkg/costmodel/costmodel.go:2624 +0x4d
github.com/kubecost/cost-model/pkg/costmodel.(*CostModel).costDataRange(0xc0003dd9e0, 0x24b6a20, 0xc000523e80, 0x251d5a0, 0xc0000c4000, 0x2510400, 0xc0003c7720, 0xc0005a4020, 0x18, 0xc0005a4060, ...)
	/app/cost-model/pkg/costmodel/costmodel.go:2030 +0x2f06
github.com/kubecost/cost-model/pkg/costmodel.(*CostModel).ComputeCostDataRange.func1(0x41300b, 0xc000a87030, 0xa4b01205, 0x320dbf3575143746)
	/app/cost-model/pkg/costmodel/costmodel.go:1705 +0x135
golang.org/x/sync/singleflight.(*Group).doCall(0xc00057b1b0, 0xc000ca2180, 0xc000faaf00, 0x2e, 0xc000a87100)
	/go/pkg/mod/golang.org/x/sync@v0.0.0-20190423024810-112230192c58/singleflight/singleflight.go:97 +0x2e
golang.org/x/sync/singleflight.(*Group).Do(0xc00057b1b0, 0xc000faaf00, 0x2e, 0xc000a87100, 0xc000ceb078, 0x2, 0x0, 0x0, 0x0)
	/go/pkg/mod/golang.org/x/sync@v0.0.0-20190423024810-112230192c58/singleflight/singleflight.go:67 +0x1ba
github.com/kubecost/cost-model/pkg/costmodel.(*CostModel).ComputeCostDataRange(0xc0003dd9e0, 0x24b6a20, 0xc000523e80, 0x251d5a0, 0xc0000c4000, 0x2510400, 0xc0003c7720, 0xc0005a4020, 0x18, 0xc0005a4060, ...)
	/app/cost-model/pkg/costmodel/costmodel.go:1704 +0x2ca
main.ComputeAggregateCostModel(0x24b6a20, 0xc000523e80, 0x20b7c55, 0x2, 0x20b7c59, 0x2, 0x20bfbd1, 0x9, 0x35044a0, 0x0, ...)
	/app/kubecost-cost-model/aggregation.go:1176 +0x1516
main.warmAggregateCostModelCache.func1(0x20b7c55, 0x2, 0x20b8037, 0x3, 0x20b7c59, 0x2)
	/app/kubecost-cost-model/main.go:493 +0x44a
main.warmAggregateCostModelCache.func2(0x2183040, 0xc000307d18)
	/app/kubecost-cost-model/main.go:520 +0x95
created by main.warmAggregateCostModelCache
	/app/kubecost-cost-model/main.go:513 +0x9c
```

**Comment by dwbrown2** [comment]
@AjayTripathy can you confirm this is resolved in v1.57.1?

**Comment by AjayTripathy** [comment]
Yes, this is resolved in 1.57.1
